### ISSUE TITLE
Splashscreen shows black but nothing

### ISSUE THREAD

**Issue by suki-neithnet** [Jigsaw-Code/outline-apps#1839]
Splashscreen shows black but nothing, it worked correctly before.

**Comment by suki-neithnet** [comment]
No, I'm running on iphone 8 with iOS 16.7.5. The splashscreen disappeared after project add appkit. But it worked correctly in previous version.

**Comment by suki-neithnet** [comment]
Sorry, pologize for the unclear description. It's the previous outlient client that worked correctly. So, maybe there's somethings wrong with the upgrade.

**Comment by sbruens** [comment]
We're closing this issue due to inactivity. If you're still experiencing problems or have any further questions, please feel free to reopen this issue or create a new one.
